__tests__/components/badges/RewardBadges.test.tsx

```diff
@@ -0,0 +1,30 @@
+import "@testing-library/jest-dom";
+import RewardBadge from "@/components/badges/RewardBadge";
+import { render, screen } from "@testing-library/react";
+import Coin from "@/components/ui/Coin";
+
+describe("RewardBadges", () => {
+  it("should render  rewardBadges", () => {
+    render(<RewardBadge />);
+    const rewardBadge = screen.getByTestId("RewardBadge");
+    expect(rewardBadge).toBeInTheDocument();
+  });
+
+  it("should render rewardBadges with coins", () => {
+    render(<RewardBadge />);
+    render(<Coin />);
+    const rewardBadge = screen.queryByTestId("RewardBadge");
+    const coin = screen.getByTestId("coin");
+    expect(rewardBadge).toBeInTheDocument();
+    expect(coin).toBeInTheDocument();
+  });
+  it("should render rewardBadges with amount and token", () => {
+    render(<RewardBadge reward={{ token: "BTC", amount: 1000 }} displayAmount={true} />);
+    const rewardBadge = screen.getByTestId("RewardBadge");
```

Comment:
you can change this test like this to check the amount using `getByText`
```suggestion
const text = screen.getByText('shortened-1000 BTC')
    expect(text).toBeInTheDocument()
```
'shortened-1000 BTC' is returned by the mocked function 

`jest.mock("../../../src/utilities", () => ({
  shortenNumber: (num:number) => `shortened-${num}`
}))`